linode/manager
# fix: [M3-7528] - Account Cancellation Survey Button Color Issues

## Description 📝
The original bug reported was that the action button on the Account Cancellation page would change from green to blue when the user would hover over it. However, an additional bug was discovered in which the Akamai logo would not display properly in Light Mode. This PR addresses both issues by updating the color of the action button to be consistent with the rest of the application, as well as its state behaviors in Light and Dark modes, and displaying the Akamai log in color when the user is in Light Mode.

## Changes  🔄
- Removes CSS styling which caused the original reported bug.
- Introduces logic to show the appropriate Akamai logo depending on the user theme preferences.

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![before-normal-light](https://github.com/user-attachments/assets/e16c59f7-19e0-45bb-ae54-2f57b09b8872) | ![after-normal-light](https://github.com/user-attachments/assets/c4625304-9d00-4a46-a559-c844b733dffe) |
| ![before-on-hover-light](https://github.com/user-attachments/assets/b1a36da0-4b28-4b40-9373-87253e569a5d) | ![after-on-hover-light](https://github.com/user-attachments/assets/0e0d1283-862f-420f-bd8e-a7db8174d82e) |
| ![before-normal-dark](https://github.com/user-attachments/assets/6c202222-5c20-4066-88a4-c6be709cd1cf) | ![after-normal-dark](https://github.com/user-attachments/assets/8d86194f-76e6-435a-bee8-fb8b64cca52a) |
| ![before-on-hover-dark](https://github.com/user-attachments/assets/32af4cc8-216c-4f78-92c7-8c13cf8fce2a) | ![after-on-hover-dark](https://github.com/user-attachments/assets/996f183d-a003-4ed4-b9ae-17641091e60e) | 

## How to test 🧪
### Prerequisites
(How to setup test environment)
- Run the following command `yarn cy:run -s "cypress/e2e/core/account/account-cancellation.spec.ts"`


### Verification steps
(How to verify changes)
- Observe the Exit Survey modal that appears as the tests run and verify that the changes are consistent with the details stated above.
- Once the tests are finished, hover over the command outputs on the Cypress UI to replay the test and verify the hover state behavior of the button.

<details>
<summary> Author Checklists </summary>

## As an Author, to speed up the review process, I considered 🤔

👀 Doing a self review
❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
🤏 Splitting feature into small PRs
➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
🧪 Providing/improving test coverage
 🔐 Removing all sensitive information from the code and PR description
🚩 Using a feature flag to protect the release
👣 Providing comprehensive reproduction steps
📑 Providing or updating our documentation
🕛 Scheduling a pair reviewing session
📱 Providing mobile support
♿  Providing accessibility support

<br/>


## As an Author, before moving this PR from Draft to Open, I confirmed ✅


</details>
